<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Reinhold Kainhofer &lt;reinhold@kainhofer.com&gt;</author>
 <comment>Configuration page for the print week mode.</comment>
 <class>CalPrintWeekConfig_Base</class>
 <widget class="QWidget" name="CalPrintWeekConfig_Base">
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Print week options:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mDateRangeGroup">
     <property name="title">
      <string>Date &amp;&amp; Time Range</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="mFromDateLabel">
        <property name="toolTip">
         <string>Starting date for the print</string>
        </property>
        <property name="whatsThis">
         <string>If you want to print more days at once, you can define a range of dates with this option and the &lt;i&gt;End date&lt;/i&gt; option. This option is used to define the start date.</string>
        </property>
        <property name="text">
         <string>&amp;Start date:</string>
        </property>
        <property name="buddy">
         <cstring>mFromDate</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KDateComboBox" name="mFromDate">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Starting date for the print</string>
        </property>
        <property name="whatsThis">
         <string>If you want to print more days at once, you can define a range of dates with this option and the &lt;i&gt;End date&lt;/i&gt; option. This option is used to define the start date.</string>
        </property>
        <item>
         <property name="text">
          <string>2009-01-19</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="mFromTimeLabel">
        <property name="toolTip">
         <string>Starting time for the print</string>
        </property>
        <property name="whatsThis">
         <string>It is possible to print only those events which are inside a given time range. With this time selection box you can define the start of this time range. The end time should be defined with the &lt;i&gt;End time&lt;/i&gt; option. Note you can automatically modify these settings if you check &lt;i&gt;Extend time range to include all events&lt;/i&gt;.</string>
        </property>
        <property name="text">
         <string>Start &amp;time:</string>
        </property>
        <property name="buddy">
         <cstring>mFromTime</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QTimeEdit" name="mFromTime">
        <property name="toolTip">
         <string>Starting time for the print</string>
        </property>
        <property name="whatsThis">
         <string>It is possible to print only those events which are inside a given time range. With this time selection box you can define the start of this time range. The end time should be defined with the &lt;i&gt;End time&lt;/i&gt; option. Note you can automatically modify these settings if you check &lt;i&gt;Extend time range to include all events&lt;/i&gt;.</string>
        </property>
        <property name="displayFormat">
         <string>hh:mm</string>
        </property>
        <property name="time">
         <time>
          <hour>8</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mToDateLabel">
        <property name="toolTip">
         <string>Ending date for the print</string>
        </property>
        <property name="whatsThis">
         <string>If you want to print more days at once, you can define a range of dates with this option and the &lt;i&gt;Start date&lt;/i&gt; option. This option is used to define the end date.</string>
        </property>
        <property name="text">
         <string>&amp;End date:</string>
        </property>
        <property name="buddy">
         <cstring>mToDate</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KDateComboBox" name="mToDate">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Ending date for the print</string>
        </property>
        <property name="whatsThis">
         <string>If you want to print more days at once, you can define a range of dates with this option and the &lt;i&gt;Start date&lt;/i&gt; option. This option is used to define the end date.</string>
        </property>
        <item>
         <property name="text">
          <string>2009-01-19</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="mToTimeLabel">
        <property name="toolTip">
         <string>Ending time for the print</string>
        </property>
        <property name="whatsThis">
         <string>It is possible to print only those events which are inside a given time range. With this time selection box you can define the end of this time range. The start time should be defined with the &lt;i&gt;Start time&lt;/i&gt; option. Note you can automatically modify these settings if you check &lt;i&gt;Extend time range to include all events&lt;/i&gt;.</string>
        </property>
        <property name="text">
         <string>End ti&amp;me:</string>
        </property>
        <property name="buddy">
         <cstring>mToTime</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QTimeEdit" name="mToTime">
        <property name="toolTip">
         <string>Ending time for the print</string>
        </property>
        <property name="whatsThis">
         <string>It is possible to print only those events which are inside a given time range. With this time selection box you can define the end of this time range. The start time should be defined with the &lt;i&gt;Start time&lt;/i&gt; option. Note you can automatically modify these settings if you check &lt;i&gt;Extend time range to include all events&lt;/i&gt;.</string>
        </property>
        <property name="displayFormat">
         <string>hh:mm</string>
        </property>
        <property name="time">
         <time>
          <hour>18</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QCheckBox" name="mIncludeAllEvents">
        <property name="toolTip">
         <string>Check this option to automatically determine the required time range, so all events will be shown.</string>
        </property>
        <property name="text">
         <string>E&amp;xtend time range to include all events</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mPrintType">
     <property name="title">
      <string>Print Layout</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="mPrintTypeFilofax">
        <property name="toolTip">
         <string>The Filofax format prints the information for the days selected without a timeline.</string>
        </property>
        <property name="text">
         <string>Print date &amp;range as Filofax format, one week per page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mPrintTypeTimetable">
        <property name="toolTip">
         <string>The timetable print view has the times to the left of the page.</string>
        </property>
        <property name="text">
         <string>Print as timeta&amp;ble view, one week per page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mPrintTypeSplitWeek">
        <property name="toolTip">
         <string>The split week view has the times to the left of the page, with two pages per week.</string>
        </property>
        <property name="text">
         <string>Print as split wee&amp;k view</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mSecurity">
     <property name="title">
      <string>Security Exclusions</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="mExcludeConfidential">
        <property name="toolTip">
         <string>Check this option to exclude items that have their Access level set to &#8220;Confidential&#8221;</string>
        </property>
        <property name="text">
         <string>Exclude c&amp;onfidential</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mExcludePrivate">
        <property name="toolTip">
         <string>Check this option to exclude items that have their Access level set to &#8220;Private&#8221;</string>
        </property>
        <property name="text">
         <string>Exclude pri&amp;vate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mIncludeInfoGroup">
     <property name="title">
      <string>Include Information</string>
     </property>
     <layout class="QGridLayout" name="includeInfoLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="mIncludeDescription">
        <property name="toolTip">
         <string>Print item descriptions</string>
        </property>
        <property name="whatsThis">
         <string>Check this option if you want to see the item descriptions printed.</string>
        </property>
        <property name="text">
         <string>&amp;Descriptions</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="mIncludeCategories">
        <property name="toolTip">
         <string>Print item tags</string>
        </property>
        <property name="whatsThis">
         <string>Check this option if you want to see the item tags printed.</string>
        </property>
        <property name="text">
         <string>Ta&amp;gs</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="mIncludeTodos">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Print to-dos due within the specified date range</string>
        </property>
        <property name="whatsThis">
         <string>You should check this option if you want to print to-dos which are due on one of the dates which are in the supplied date range.</string>
        </property>
        <property name="text">
         <string>Include to-&amp;dos that are due on the printed day(s)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="mExcludeTime">
        <property name="toolTip">
         <string>Check this option to exclude the time in description box</string>
        </property>
        <property name="text">
         <string>Exclude t&amp;ime from timetable detail display items</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mGeneralGroup">
     <property name="title">
      <string comment="@title general print settings">General</string>
     </property>
     <layout class="QVBoxLayout" name="generalLayout">
      <item>
       <widget class="QCheckBox" name="mSingleLineLimit">
        <property name="toolTip">
         <string>Print items on one line</string>
        </property>
        <property name="whatsThis">
         <string>Check this option to limit events to a single line, truncating as necessary to save space.</string>
        </property>
        <property name="text">
         <string>Limit events in each day to a &amp;single line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mShowNoteLines">
        <property name="toolTip">
         <string>Check this option to draw note lines in empty areas.</string>
        </property>
        <property name="text">
         <string>Show note &amp;lines in Filofax layout</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mColors">
        <property name="toolTip">
         <string>Print in color</string>
        </property>
        <property name="whatsThis">
         <string>If you want to use colors to distinguish certain tags in the printed output, check this option.</string>
        </property>
        <property name="text">
         <string>&amp;Use colors</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mPrintFooter">
        <property name="toolTip">
         <string>Print a datetime footer on each page</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to print a small footer on each page that contains the date of the print.</string>
        </property>
        <property name="text">
         <string>Print &amp;Footer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDateComboBox</class>
   <extends>QComboBox</extends>
   <header>kdatecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
